src/ext/ca/wixca/dll/qtexecca.cpp

```diff
@@ -62,17 +82,17 @@ HRESULT BuildCommandLine(
 
 #define ONEMINUTE 60000
 
-DWORD GetTimeout()
+DWORD GetTimeout(LPCWSTR wzPropertyName)
 {
     DWORD dwTimeout = ONEMINUTE;
     HRESULT hr = S_OK;
 
     LPWSTR pwzData = NULL;
 
-    if (WcaIsPropertySet("QtExecCmdTimeout"))
+    if (WcaIsUnicodePropertySet(wzPropertyName))
     {
-        hr = WcaGetProperty( L"QtExecCmdTimeout", &pwzData);
-        ExitOnFailure(hr, "failed to get QtExecCmdTimeout");
+        hr = WcaGetProperty(wzPropertyName, &pwzData);
+        ExitOnFailure1(hr, "failed to get %ls", wzPropertyName);
```

Comment:
Should use the variadic macros now (just drop the "1" in the macro - they will be removed eventually).